packages/core/components/Puck/index.tsx

```diff
@@ -401,6 +413,32 @@ function PuckLayout<
   );
 
   const dispatch = useAppStore((s) => s.dispatch);
+  const appStoreApi = useAppStoreApi();
+
+  // Add delete hotkey functionality
+  const handleDelete = useCallback(() => {
```

Comment:
Can you wrap this up inside the `useDeleteHotkeys` hook, and move that hook to `lib/use-delete-hotkeys.ts`? I want to avoid bloating the Puck component here.